<ui version="4.0" >
 <author>Aaron Racicot</author>
 <class>NextPolygon</class>
 <widget class="QDialog" name="NextPolygon" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Next Polygon</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QWidget" native="1" name="widget_2" >
     <widget class="QLabel" name="label_1" >
      <property name="geometry" >
       <rect>
        <x>8</x>
        <y>9</y>
        <width>123</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string>Weighting (Pennies)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_1" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>137</x>
        <y>9</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pl_label" >
      <property name="geometry" >
       <rect>
        <x>510</x>
        <y>0</y>
        <width>50</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize" >
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pbnShapeFinished" >
        <property name="text" >
         <string>Finished With Fishery</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnShapeDiscard" >
        <property name="text" >
         <string>Discard Last Shape...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbnMoreShapes" >
        <property name="text" >
         <string>More Shapes This Fishery...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
